### PR TITLE
Add AncientPotato.

### PR DESCRIPTION
**Pull request by razorordeath**
Added ancient potatoes and 2 extra delicious condiments.

### CODE REVIEW

File: src/main/java/org/drtshock/Potato.java [hunk 1]
```diff
@@ -87,6 +88,9 @@ public boolean isPutIntoOven() throws OvenException {
         }
     }
 
+    public void isAncientPotato() {
```
Comment (jkcclemens) [inline src/main/java/org/drtshock/Potato.java]: This should return a `boolean`. If you want there to be a message, consider throwing an exception.

File: src/main/java/org/drtshock/Potato.java [hunk 2]
```diff
@@ -20,6 +20,7 @@ public static void main(String[] args) {
             System.out.println("Of course potato is prepared and delicious.");
         } catch (NotDeliciousException e) {
             System.err.println("Fatal error! How could potato not be delicious?");
+
```
Comment (jkcclemens) [inline src/main/java/org/drtshock/Potato.java]: Unnecessary.

**Comment by jkcclemens** [comment]
Personally, I think adding mayonnaise AND mustard to the crazy amount of condiments we already have would result in a non-tasty potato.

**Comment by Ichbinjoe** [comment]
There is also an open ended method... Shouldn't there be an actual check?

**Comment by drtshock** [comment]
I don't understand why we are doing isAncientPotato then just printing a static string.

Also, mayo on a potato? :-1:

**Comment by jkcclemens** [comment]
No activity for almost a month. Closing.

**Pull request closed without merging**